src/app/components/TimeToEpoch/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { selectEpoch } from 'app/state/network/selectors'
+import { useSelector } from 'react-redux'
+import * as React from 'react'
+
+interface Props {
+  epoch: number
+}
+
+const estimatedEpochsPerHour = 1
+
+const relativeFormat = new Intl.RelativeTimeFormat(process?.env?.NODE_ENV === 'test' ? 'en-US' : undefined)
```

Comment:
The downside is that we loose multi lang support. It will show EN labels no matter what lang user is using.

Reply:
in production `new Intl.RelativeTimeFormat(undefined).format(1, 'day')` it should be localized:
![image](https://user-images.githubusercontent.com/3758846/183772908-9111a25d-0856-46b9-a193-8b3bbf5b3f18.png)

Reply:
Either way it's not in sync with app lang. Mixing langs is a bit weird, but in this case it is still better than epoch number.